test/testcafe/spec/TerminalView_spec.js

```diff
@@ -173,3 +178,20 @@ async function setupOtpOnly(t) {
               
     });
 });
+
+[  
+  ['should have biometrics required error in loopback polling page', terminalBiometricsReqruiedForFastPassMock],
```

Comment:
Add the test in it's own view specific testcafe file.. Do not add in TerminalView